doc/ja/guides/strengths.md

```
@@ -101,29 +101,37 @@ tmaiは「監視に徹する」ため、ワークフローを強制しません
 
 詳細: [ワークツリーで並列開発](../workflows/worktree-parallel.md)
 
-## 5. PTYラッピングによる高精度検出
+## 5. 3段構えの高精度検出
 
-`tmai wrap claude` で起動すると、PTYプロキシ経由で入出力を直接監視。
+tmaiは最大の精度を実現するために3段構えの検出戦略を使用：
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)